### ISSUE TITLE
Accessibility Audit: Heading level 1 not available under login/account pages

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#941]
Turn on the Screen Reader and go to the [Login](https://sites-public-tj3gg4i5eq-uc.a.run.app/sign-in) page, followed by the [Account Dashboard](https://sites-public-tj3gg4i5eq-uc.a.run.app/account/dashboard) page and [Account Settings](https://sites-public-tj3gg4i5eq-uc.a.run.app/account/edit) page. None of these pages have a heading Level 1. These pages should have a heading level 1 and headings in the page should be in a hierarchical order.

https://user-images.githubusercontent.com/394908/147792076-b5d97e90-2cae-4ff5-a0ea-fb503754efc5.mov

**Comment by emilyjablonski** [comment]
Fixed in Seeds
